src/invenio-files-js/controllers/InvenioFilesCtrl.js

```diff
@@ -269,7 +269,7 @@ function InvenioFilesCtrl($rootScope, $scope, $q, $timeout,
     * @function fileUplaodedError
     */
   function fileUploadedError(evnt, data) {
-    var index = findInFiles(data.config.data.file.name);
+    var index = findInFiles(data.config.data.file.name || data.config.data.name);
```

Comment:
``var index = findInFiles(data.config.data.file !== undefined ? data.config.data.file.name : data.config.data.name);``

Otherwise it doesn't show the error message.